docs/4-user-guide/4-vscode.md

```diff
@@ -0,0 +1,26 @@
+# Getting started - VS Code
+
+Zarf uses an internally defined [schema](https://github.com/defenseunicorns/zarf/blob/master/zarf.schema.json) for its configuration files. This schema is used to validate the configuration files before they are used to build a Zarf package.
+
+## Adding Schema validation
+
+1. Open VS Code's `settings.json` file with `CTRL/CMD + SHIFT + P` and search for `Preferences: Open User Settings (JSON)`.
```

Comment:
it should be noted that you need the yaml extension installed in VSCode for this to work (the one made by Red Hat)